lib/vector-buffers/src/topology/channel/sender.rs

```diff
@@ -295,7 +295,11 @@ impl<T: Bufferable> Sink<T> for BufferSender<T> {
             // We've been instructed to drop the next item.
             SendState::DropNext => {
                 if let Some(instrumentation) = this.instrumentation.as_ref() {
-                    if let Some((item_count, _)) = item_sizing {
+                    if let Some((item_count, item_size)) = item_sizing {
+                        instrumentation.increment_received_event_count_and_byte_size(
```

Comment:
Can we include a note in the upgrade guide for this change?

Reply:
Ah, doh, yes I had that thought and then completely forgot 😅